.buildkite/hooks/pre-exit

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+if [[ $ENABLE_STEP_CDT == '1' && -x bin/task ]]; then
+  bin/task ci:cdt-cleanup
```

Comment:
`ENV` is not needed?

Reply:
yes, `ENV` not needed. it is not used in the task or the sub-tasks so i got rid of it from previous commit.